deployment/terraform/create_utils.go

```diff
@@ -35,3 +36,97 @@ func (t *Terraform) setupPrometheusNodeExporter(sshClient *ssh.Client) error {
 
 	return nil
 }
+
+// modifyRXSize modifies the size of the receiving buffer of the network card
+// to the maximum permitted by the hardware.
+func modifyRXSize(sshc *ssh.Client) error {
+	rxSizes, err := getEthtoolRxSizes(sshc)
+	if err != nil {
+		return fmt.Errorf("error getting the RX sizes from the ethtool: %w", err)
+	}
+
+	oldRxSize := rxSizes.actualRX
+
+	// Modify RX queue size to the maximum permitted
+	incRxSizeCmd := fmt.Sprintf("sudo ethtool -G $(ip route show to default | awk '{print $5}') rx %d", rxSizes.maxRX)
+	cmd := fmt.Sprintf("%s && sudo sysctl -p && sudo systemctl restart nginx", incRxSizeCmd)
```

Comment:
Since we are always setting the max queue size, I wonder if we can come up with a single unix one-liner which parses the maxRX from the `-g` command, and feeds that to the `-G` command? Or if that's cumbersome, this is fine as well. 0/5